client/commonFramework/output-display.js

```diff
@@ -7,6 +7,20 @@ window.common = (function(global) {
 
   const { challengeTypes, challengeType = '0' } = common;
 
+  function convertDisplayValue(displayVal) {
+    var convertedDisplayVal = displayVal;
+    var convertedArray = [];
+    if (displayVal instanceof Array) {
+      convertedDisplayVal = `["${displayVal.join('", "')}"]`;
+    } else if (displayVal !== null && typeof displayVal === 'object') {
+      convertedArray = Object.keys(displayVal).map((key) =>
+        '' + displayVal[key]
+      );
+      convertedDisplayVal = `["${convertedArray.join('", "')}"]`;
+    }
+    return convertedDisplayVal;
+  }
```

Comment:
This function could benefit from using the return early pattern https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/The-Return-Early-Pattern